Tools/LyTestTools/tests/unit/test_fixtures.py

```diff
@@ -187,13 +187,6 @@ def test_Launcher_MockHelper_Passthrough(self, mock_create):
         retval = mock.MagicMock()
         mock_create.return_value = retval
         mock_workspace = mock.MagicMock()
-        mock_workspace.paths.waf.return_value = "dummy"
```

Comment:
This is the danger of over-mocking an object. Waf hasn't been used in years at this point, and this was silently doing nothing!